src/languageProvider/locales/en_US.json

```diff
@@ -123,6 +123,7 @@
   "str.m": "m",
   "str.minutes": "Minutes",
   "str.name": "Name",
+  "str.newBlock": "New block comes, you can refresh to get new data",
```

Comment:
is this the message when you can load more data?

Reply:
it's for new block coming, refresh to load the new data, not for load more, it's for load new

Reply:
Does the user need to manually refresh to get the data? Like have to push the refresh button? Because I took out the refresh capability in the electron version. Refreshing shouldnt be needed as react should handle all incoming changes.